sources/packages/backend/apps/api/src/route-controllers/cas-supplier/_tests_/cas-supplier.aest.controller.getCASSupplier.e2e-spec.ts

```diff
@@ -0,0 +1,70 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  saveFakeCASSupplier,
+} from "@sims/test-utils";
+import { SupplierStatus } from "@sims/sims-db";
+
+describe("CASSupplierAESTController(e2e)-getCASSuppliers", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should get all the CAS suppliers for a student when requested given a student id.", async () => {
```

Comment:
Minor, IMO the `requested` can be removed. It made the sentence harder to read.

Reply:
I rephrased the sentence.